Meziantou.Xunit.ParallelTestFramework.Tests/TestOutputHelperTests.cs

```diff
@@ -1,5 +1,7 @@
 using Xunit;
+#if !XUNIT_V3
 using Xunit.Abstractions;
+#endif

```

Comment:
I decided to reuse the test code as only this one line changes between them (perhaps this could be a project `Using` to side-step the the if-def)